notebooks/From FHIR to ML - Getting Started with the PHC Python SDK.ipynb

```diff
@@ -0,0 +1,2264 @@
+{
```

Comment:
<p>If this is going in an example I'd be happier with <code>phc&gt;=0.23.2</code>  instead of pinning to this version. Are you concerned about future breaking changes here? If so, maybe add a note about the difference between <code>==</code>  and <code>&gt;=</code>  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lifeomic/phc-examples/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>